### ISSUE TITLE
Textfield issues and incorrect textfield applied in tabview

### ISSUE THREAD

**Issue by XANOZOID** [haxeui/haxeui-kha#63]
I have created a test app. Basically it comes in 2 parts login and create account. They're very similar. I have also attached a GIF of what happens. Currently, textfields may lag on input (as demonstrated twice in the gif on the password fields - but I've gotten regular textfields doing the same thing before). Then if you notice, I'm typing on the login page but the textfields from the create account page are the ones receiving the text. 

The third problem is most obvious. If you click the login button I removed the whole component from the screen. If you start typing any letters or left/right arrows the textfields are apparently still hooked and throw an exception - you must be debugging and catching uncaught exceptions to see this error.

The fourth problem, not displayed, is that if you keep pressing backspace in an empty textfield you essentially break its positional information. Once you've pressed backspace enough times, try entering in some text - it will all be entered at the front of the character list instead of the back like normal. Using the arrows to correct it does not work.

So that's 4 problems
1: text field lag (slightly inconsistent).
2: selecting text fields in other tab view pages (fairly consistent).
3: text field events remain registered when their parents are removed from the tree. (consistent)
4: Pressing backspace repeatedly breaks typing in the text fields. (consistent)

## Media
![haxeuiexample](https://user-images.githubusercontent.com/29788070/235234964-48971ccc-6ebd-431b-84af-19ba71f17573.gif)

## Test app / minimal test case
Attached  is a similar test app to my last issue but with more haxeui components.
[testapp.zip](https://github.com/haxeui/haxeui-kha/files/11356723/testapp.zip)

## Context
Unresponsive/broken text fields make for an unpleasant user experience.

## Your Environment
* Version used: latest
* Environment name and version (e.g. Chrome 39, node.js 5.4): Chrome Debugger
* Operating System and version (desktop or mobile): Windows 10

**Comment by ianharrigan** [comment]
Can you pull latest and retry? There were some oddities which i think have been fixed now.

Cheers,
Ian

**Comment by XANOZOID** [comment]
Pulled latest haxeui-core (from haxelib) and pulled latest haxeui-kha (git). 

Still experiencing the following: 
Point 1: There is still, random, chances where lag occurs when typing. It's like, when I first start typing I see nothing populate the text box. If I finish typing (while nothing is printing), then click the textbox again, it fills with what I typed (after the fact).

Point 3: Still in a situation where removing an element that contains the textbox somewhere down the component tree the event listeners don't appear to be unhooking. It may be that I'm removing the container element while the textbox is an active element? Like, within the button click event I remove the entire container while the textbox was active before I clicked the button that removes the container. It's possible to test if events weren't unhooked by debugging for uncaught exceptions by pressing character/digit keys after removing the container.

Point 4: It is still happening where if you hold the backspace or repeatedly tap it while the textbox is empty that the typing of the textbox becomes broken.

Thanks!
